docs/tutorial/CustomDataLoaderOnSL.ipynb

```diff
@@ -76,108 +95,81 @@
    "metadata": {},
    "source": [
-    "我们定义DataBuilder的目的是已有的FedDataFrame和FedNdarray提供的功能无法满足需求，我们可以在通过自定义DataBuilder来满足高阶定制化需求。  \n",
-    "在SplitLearning中（各方数据需要对齐）我们暂时仅提供了面向CSV数据的DataBuilder能力。在自定义DataBuilder中可以定义对每一行数据怎么处理。SLModel会根据您定义的方式进行操作。\n",
+    "We customize a DataBuilder with the aim to meet higher level customization demands, as the existing FedDataFrame and FedNdarray did not provide the required functionality.\n",
     "\n",
-    "需要注意的点:  \n",
-    "- 我们的MovieLens数据集读取进来后会是CSV格式的，但是需要被当做稀疏特征来进行处理，所以会在DataBuilder中将每一列转成字典形式\n",
-    "- 在Bob测我们将评分进行了Threshold处理成二值形式\n",
-    "- 在DataBuilder中可以定义自定义函数，然后使用dataset.map来应用到每一列。\n",
-    "- 因为垂直模式需要保证两方数据是对齐的，所以暂时只支持CSV模式\n",
-    "- CSV模式只需要返回Dataset，Dataset需要定义好Batch_size，Repeat，SLModel会根据Dataset推断Steps_per_epoch"
+    "In SplitLearning(where all sides of the data need to be aligned), we only provide DataBuilder for CSV data for the time being. You can define what to do with each row of data in the custom DataBuilder. The operations in SLModel depend on the way you define it.\n",
+    "\n",
+    "The things you should notice here:\n",
+    "- We read MovieLens dataset in CSV format. However, it needs to be treated as sparse features. We should convert each column in the DataBuilder to dictionary form.\n",
+    "- Bob's score was processed into binary form using threshold.\n",
+    "- Custom functions can be defined in the DataBuilder and then applied to each column using dataset.map.\n",
+    "- Only CSV mode is supported in the vertical mode due to the restriction that both sides of the data should be aligned. \n",
+    "- It returns Dataset which has been defined Batch_size and Repeat in the CSV mode so that SLModel can infer the Steps_per_epoch according to the Dataset.\n",
+    "\n"
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "e79c519b",
    "metadata": {},
    "source": [
-    "### 1、下载数据，并转换成csv"
+    "### 1. Download the dataset and convert it to a CSV format."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
-   "id": "2a333a2c",
+   "execution_count": 6,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "E0407 17:58:18.231911774  122073 fork_posix.cc:76]           Other threads are currently calling into gRPC, skipping fork() handlers\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "%%capture\n",
     "%%!\n",
     "wget https://secretflow-data.oss-accelerate.aliyuncs.com/datasets/movielens/ml-1m.zip\n",
     "unzip ./ml-1m.zip "
    ]
   },
-  {
-   "attachments": {},
-   "cell_type": "markdown",
-   "id": "483bfc93",
-   "metadata": {},
-   "source": [
-    "读取dat格式的数据并转成字典"
-   ]
-  },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "id": "6d050edf",
+   "execution_count": 9,
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Read the data from the dat format and convert it to a dictionary form\n",
```

Comment:
No `#`
`convert it to a dictionary form` -> `convert it to the dictionary form`